code/controllers/subsystem/shuttles/emergency.dm

```diff
@@ -105,11 +105,14 @@
 				priority_announce("Hostile enviroment detected. Departure has been postponed indefinitely pending conflict resolution.", null, 'sound/misc/notice1.ogg', "Priority")
 				mode = SHUTTLE_STRANDED
 			if(time_left <= 0 && !SSshuttle.emergencyNoEscape)
+				world << 'sound/effects/hyperspace_begin.ogg'
+				sleep(55) //I TRIED PUTTING A SPAWN() HERE AND IT MADE THE SHUTTLE EXPLODE. DON'T TOUCH THIS. Also it's 55 so it times up with the launch.
 				//move each escape pod to its corresponding transit dock
 				for(var/obj/docking_port/mobile/pod/M in SSshuttle.mobile)
 					if(M.z == ZLEVEL_STATION) //Will not launch from the mine/planet
 						M.enterTransit()
 				//now move the actual emergency shuttle to its transit dock
+				world << 'sound/effects/hyperspace_progress.ogg'
```

Comment:
Actually I didn't notice this but the people on the station shouldn't be hearing all the stages, that makes no sense